docs/pact-reference.md

```diff
@@ -740,7 +738,7 @@ arguments in order to serially execute the function.
 
 ```lisp
 (map (+ 2) [1 2 3])
-(fold (+) ["Concatenate" " " "me"]
+(fold (+) "" ["Concatenate" " " "me"]
```

Comment:
Probably an oversight, unless previous versions of `fold` functioned like Haskell's `foldl1`.

Reply:
Are we missing a parenthesis on line 741?